<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>euclideanw</class>
 <widget class="QWidget" name="euclideanw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>输入N:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>欧几里得算法：求gcd</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberN">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberM">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>结果是：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="numberResult">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="endExcute">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>输入M:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="excute">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
